fares-data-build-tool/fdbt-site
# Feature/tfn 927 service option single service

# Description

Add the service end day option on the single period journey

# Testing instructions

- Browse to the /periodValidty page and test the scenairos
- Also as there has been a change to the radioConditional component it would be worth testing definePassengerType and defineTimeRestrictions page to see if they work correctly

# Type of change


# Checklist:
